package-res/settings.xml

```diff
@@ -12,5 +12,10 @@
     <auto-submit>false</auto-submit>
     <auto-submit-default>false</auto-submit-default>
 	-->
+
+    <!-- Enabled or disabled row limit control on UI. No value means disabled.-->
+    <query-limit-enabled>false</query-limit-enabled>
+    <!-- The maximum number of rows that will be rendered in a report viewer. -->
+    <query-limit>100000</query-limit>
     
```

Comment:
This is a bit misleading. We have at least two limits here:

(a) System-limit - the hard limit set by the administrator. No one can ever circumvent that. (Oh, and to be backward compatible, it has be *disabled* by default.)
(b) user-limit - the a default value used for the UI. Can be lower than the system-limit. This value would be used if the report itself does not define a value. 

But whether we should use the report's defined query-limit at all as a default value is another topic for tomorrows meeting. 

I would also rename the "query-limit-enabled" flag to "query-limit-ui-enabled" to make it clear that this flag controls the UI and user-limit, but does not enable/disable the system-limit.